lib/id3c/cli/command/etl/manifest.py

```diff
@@ -206,9 +227,9 @@ def upsert_sample(db: DatabaseSession,
         LOG.info(f"Updating existing sample {sample.id}")
         sample = db.fetch_row("""
             update warehouse.sample
-               set identifier = %(identifier)s,
-                   collection_identifier = %(collection_identifier)s,
-                   collected = date_or_null(%(collection_date)s),
+               set identifier = coalesce(%(identifier)s, identifier),
+                   collection_identifier = coalesce(%(collection_identifier)s, collection_identifier),
+                   collected = coalesce(date_or_null(%(collection_date)s), collected),
```

Comment:
See if you agree with the 3 coalesces I added here. The idea is not to wipe out a value with an update that doesn't have the field populated.

Reply:
@tsibley made a similar change in his PR #161 😄

Reply:
These changes are no longer in the new implementation, so I'm resolving this comment thread.